app/__tests__/home.test.ts

```diff
@@ -1,26 +1,7 @@
-import { UITestBuilder } from '@antiwork/shortest';
+import { test } from '@antiwork/shortest';
 
 interface loginButton {
   url: string;
 }
 
-define('Validate Dasboard is accessible by users', async () => {
-  afterAll(async () => {
-    console.log('Clearing DB after all tests');
-  });
-  beforeAll(async () => {
-    console.log('Clearing DB before all tests');
-  });
-  new UITestBuilder<loginButton>('/')
```

Comment:
Removed UITestBuilder, too complex. Also shouldn't need to have an Interface created. Humans don't think this way.